camel-sap/pom.xml

```diff
@@ -18,73 +24,367 @@
 		<version>2.17.0.redhat-630133</version>
 	</parent>
 
-	<groupId>org.fusesource</groupId>
+	<groupId>org.fusesource.components</groupId>
 	<artifactId>camel-sap-parent</artifactId>
 	<version>6.3.0.redhat-SNAPSHOT</version>
 	<packaging>pom</packaging>
 
-	<name>JBoss Fuse :: Components :: SAP JCO</name>
+	<name>JBoss Fuse :: Components :: SAP</name>
 	<description>Provides SAP Camel Component</description>
 	<url>http://www.jboss.org/products/fuse/overview/</url>
 
+	<organization>
+		<name>Red Hat</name>
+		<url>http://www.redhat.com</url>
+	</organization>
+
+	<issueManagement>
+		<system>jira</system>
+		<url>https://issues.jboss.org/browse/ENTESB</url>
+	</issueManagement>
+
+	<scm>
+		<connection>scm:git:git://git@github.com:jboss-fuse/fuse-components.git</connection>
+		<developerConnection>scm:git:ssh://git@github.com:jboss-fuse/fuse-components.git</developerConnection>
+		<url>https://github.com/jboss-fuse/fuse-components</url>
+	</scm>
+
+	<prerequisites>
+		<maven>3.6</maven>
```

Comment:
I'm wondering if it is compatible with productization requirements for 6.3 stream. Worth checking.